src/coreclr/tools/StressLogAnalyzer/StressLogDump.cpp

```diff
@@ -162,8 +179,8 @@ void formatOutput(struct IDebugDataSpaces* memCallBack, ___in FILE* file, __inou
                         // Print the string up to that point
                     c = *ptr;
                     *ptr = 0;       // Terminate the string temporarily
-                    vfprintf(file, format, (va_list)args);
-                    *ptr = c;       // Put it back
+                    fprintf(file, format, args[0], args[1], args[2], args[3], args[4], args[5], args[6], args[7], args[8], args[9], args[10]); 
```

Comment:
Why was this change needed? PAL has `vfprintf` implementation.

Reply:
The `vfprintf` function works. The problem is that on Linux, the cast from `void**` to `va_args` is forbidden.

Reply:
Ah, I haven't noticed that we are passing in args as void**. The failure to cast on Unix makes sense, varargs is not a pointer, but rather a data structure there.